src/expressions/block-expr.md

```diff
@@ -44,7 +44,7 @@ r[expr.block.result]
 Then the final operand is executed, if given.
 
 r[expr.block.type]
-The type of a block is the type of the final operand, or `()` if the final operand is omitted.
+Typically, the type of a block is the type of the final operand, or `()` if the final operand is omitted.
```

Comment:
Saying "typically" is a bit loose for the normative text.  Perhaps it'd be better to put the diverging rule first (and to mention that such a block is of the never type) and then, in this rule, to say, "Otherwise, the type of a block is...".  Or, alternative, in this rule, to say "the type of a block is... except ...".